core/src/test/java/org/apache/iceberg/TestTableMetadataSerialization.java

```diff
@@ -78,4 +82,21 @@ public void testSerialization() throws Exception {
         Lists.transform(result.snapshots(), Snapshot::snapshotId));
     Assert.assertEquals("History should match", meta.snapshotLog(), result.snapshotLog());
   }
+
+  private void assertSameSchemaList(List<Schema> list1, List<Schema> list2) {
```

Comment:
Is it possible to factor this into a common util rather than copying it?

Reply:
I didn't find a test util class in core, but looks like there's a `TestHelpers` in api module so I'll put it there.